src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/IdamCachedClientTest.java

```diff
@@ -250,7 +251,7 @@ public void should_invalidate_userDetail_when_cached_token_removed_from_cache()
         assertThat(userDetailsBefore).isEqualTo(expectedUserDetails1);
 
         idamCachedClient.removeAccessTokenFromCache(jurisdiction1);
-        idamCachedClient.cleanUpAccessTokenCache();
+        idamCachedClient.cleanUpCaches();
```

Comment:
I realise it was like that before, but does this test verify the behaviour of `removeAccessTokenFromCache` if we clean the whole cache afterwards?

Reply:
it is not cleaning the cache from javadoc of method:
  /**
   * Performs any pending maintenance operations needed by the cache. Exactly which activities are
   * performed -- if any -- is implementation-dependent.
   */

Reply:
first i tried 3 sec sleep, it passed test but it was luck.

Reply:
Not sure we need `cleanUpCaches` to exist at all

Reply:
aaa, I see